index.html

```diff
@@ -127,6 +129,7 @@ <h2>Tonic</h2>
           </div>
         </div>
       </div>
+      <!-- Mobile Project 2-->
```

Comment:
- [ ] The other reviewer requested that you display all cards dynamically. What he meant was that you should display the four cards in the project section of your portfolio through the use of javascript. You are meant to use the array of objects which you used to populate the popup to also dynamically display the cards section. You can use the same strategy you used for the popup to display this section also. 👍